apps/parsley/vite.config.ts

```diff
@@ -12,114 +12,119 @@ import path from "path";
 import { generateBaseHTTPSViteServerConfig } from "@evg-ui/vite-utils";
 import injectVariablesInHTML from "./config/injectVariablesInHTML";
 
-// Remove when https://github.com/cypress-io/cypress/issues/25397 is resolved.
-dns.setDefaultResultOrder("ipv4first");
+let finalConfig: Record<string, any> = {};
 
-const serverConfig = generateBaseHTTPSViteServerConfig({
-  port: 5173,
-  appURL: process.env.REACT_APP_PARSLEY_URL,
-  httpsPort: 8444,
-  useHTTPS: process.env.REACT_APP_RELEASE_STAGE !== "local",
-});
+// Only run this configuration if we are not in script mode
+if (process.env.VITE_SCRIPT_MODE !== "1") {
```

Comment:
Since this diff is way larger than it should be the only real change here is moving finalConfig into its own variable. and nesting the rest of the config under the conditional on L18